sys/shell/commands/shell_commands.c

```diff
@@ -214,5 +218,8 @@ const shell_command_t _shell_command_list[] = {
     { "ccnl_cont", "create content and populated it", _ccnl_content},
     { "ccnl_fib", "shows or modifies the CCN-Lite FIB", _ccnl_fib},
 #endif
+#ifdef MODULE_SNTP
+    { "ntpdate", "synchronizes with a remote time server", _ntpdate},
```

Comment:
This file is a already a mess style-wise, but also add a space after `_ntpdate`.